OuterWildsRandomSpeedrun/SpawnPointPool.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace OuterWildsRandomSpeedrun
+{
+    public struct SpawnPointConfig
+    {
+        public string internalId;
+        public string displayName;
+
+        public SpawnPointConfig(string internalId, string displayName)
+        {
+            this.internalId = internalId;
+            this.displayName = displayName;
+        }
+    }
+
+    public class SpawnPointPool
+    {
+        public List<SpawnPointConfig> SpawnPointConfigs
+        {
+            get => _spawnPointConfigs;
+        }
+
+        private List<SpawnPointConfig> _spawnPointConfigs;
+
+        private SpawnPointPool(List<SpawnPointConfig> configs)
+        {
+            this._spawnPointConfigs = configs;
+        }
+
+        public static SpawnPointPool FromTsv(string pathToTsv)
+        {
+            var configs = ParseTsv(pathToTsv).Select(line => BuildSpawnPointConfig(line)).ToList();
+            return new SpawnPointPool(configs);
+        }
+
+        private static SpawnPointConfig BuildSpawnPointConfig(string[] line)
+        {
+            var internalId = line[0];
+            var displayName = line[1];
+            return new SpawnPointConfig(internalId, displayName);
```

Comment:
you can use the default constructor with an initializer list:
`return new SpawnPointConfig {
  InternalId = internalId,
  DisplayName = displayName,
}`